modules/timepoint_list/README.md

```diff
@@ -0,0 +1,48 @@
+# Timepoint List
+
+## Purpose
+
+The `timepoint_list` module serves as an entry-point for data entry
+about a candidate. It serves as a dashboard for a candidate linking
+to other modules that provide specific functionality about a candidate.
```

Comment:
how about... `more detailed information and actions for this candidate's data `

Reply:
I don't like either the word "specific" or "detailed" but I left it as "specific" because it's slightly more accurate and took the rest of your suggestion.

Reply:
sure, works for me.